packages/backend/server/src/models/common/page.ts

```diff
@@ -1,4 +1,11 @@
-export enum PublicPageMode {
+export enum PublicDocMode {
   Page,
   Edgeless,
 }
+
+export enum WorkspaceRole {
```

Comment:
keep the pr clean

Reply:
ok, I will rewrite it after 0.20 released.